### ISSUE TITLE
Data format

### ISSUE THREAD

**Issue by mzkhan2000** [ttrouill/complex#5]
your data in three files train.txt, valid.txt and test.txt Each line is a triple, but not in the format like ubject_entity_id    relation_id	object_entity_id. How your program works with this data that stored in datasets fb15k.zip and wn18.zip file?

**Comment by ttrouill** [comment]
I am not sure I get your question, can you rephrase it?

**Comment by ttrouill** [comment]
I guess you've found your answer, I'll close the issue.
